pydicom/uid.py

```diff
@@ -211,92 +226,132 @@ def is_valid(self) -> bool:
 """1.2.840.10008.1.2.4.104"""
-MPEG4HighProfileLevel423D = UID('1.2.840.10008.1.2.4.105')
+MPEG4HP423D = UID('1.2.840.10008.1.2.4.105')
 """1.2.840.10008.1.2.4.105"""
-MPEG4StereoHighProfileLevel42 = UID('1.2.840.10008.1.2.4.106')
+MPEG4HP42STEREO = UID('1.2.840.10008.1.2.4.106')
 """1.2.840.10008.1.2.4.106"""
-HEVCMainProfileLevel51 = UID('1.2.840.10008.1.2.4.107')
+HEVCMP51 = UID('1.2.840.10008.1.2.4.107')
 """1.2.840.10008.1.2.4.107"""
-HEVCMain10ProfileLevel51 = UID('1.2.840.10008.1.2.4.108')
+HEVCM10P51 = UID('1.2.840.10008.1.2.4.108')
 """1.2.840.10008.1.2.4.108"""
 RLELossless = UID('1.2.840.10008.1.2.5')
 """1.2.840.10008.1.2.5"""
 
-UncompressedPixelTransferSyntaxes = [
-    ExplicitVRLittleEndian,
+AllTransferSyntaxes = [
     ImplicitVRLittleEndian,
+    ExplicitVRLittleEndian,
     DeflatedExplicitVRLittleEndian,
     ExplicitVRBigEndian,
-]
-
-JPEGLSSupportedCompressedPixelTransferSyntaxes = [
+    JPEGBaseline8Bit,
+    JPEGExtended12Bit,
+    JPEGLosslessP14,
+    JPEGLosslessSV1,
     JPEGLSLossless,
-    JPEGLSLossy,
-]
-
-PILSupportedCompressedPixelTransferSyntaxes = [
-    JPEGBaseline,
-    JPEGLossless,
-    JPEGExtended,
+    JPEGLSNearLossless,
     JPEG2000Lossless,
     JPEG2000,
+    JPEG2000MCLossless,
+    JPEG2000MC,
+    MPEG2MPML,
+    MPEG2MPHL,
+    MPEG4HP41,
+    MPEG4HP41BD,
+    MPEG4HP422D,
+    MPEG4HP423D,
+    MPEG4HP42STEREO,
+    HEVCMP51,
+    HEVCM10P51,
+    RLELossless,
 ]
+"""All non-retired transfer syntaxes and *Explicit VR Big Endian*."""
 
-JPEG2000CompressedPixelTransferSyntaxes = [
-    JPEG2000Lossless,
-    JPEG2000,
+JPEGTransferSyntaxes = [
+    JPEGBaseline8Bit, JPEGExtended12Bit, JPEGLosslessP14, JPEGLosslessSV1
 ]
+"""JPEG (ISO/IEC 10918-1) transfer syntaxes"""
 
-JPEGLossyCompressedPixelTransferSyntaxes = [
-    JPEGBaseline,
-    JPEGExtended,
+JPEGLSTransferSyntaxes = [JPEGLSLossless, JPEGLSNearLossless]
+"""JPEG-LS (ISO/IEC 14495-1) transfer syntaxes."""
+
+JPEG2000TransferSyntaxes = [
+    JPEG2000Lossless, JPEG2000, JPEG2000MCLossless, JPEG2000MC
 ]
+"""JPEG 2000 (ISO/IEC 15444-1) transfer syntaxes."""
 
 MPEGTransferSyntaxes = [
-    MPEG2MainProfileMainLevel,
-    MPEG2MainProfileHighLevel,
-    MPEG4HighProfileLevel41,
-    MPEG4BDCompatibleHighProfileLevel41,
-    MPEG4HighProfileLevel422D,
-    MPEG4HighProfileLevel423D,
-    MPEG4StereoHighProfileLevel42,
-    HEVCMainProfileLevel51,
-    HEVCMain10ProfileLevel51,
+    MPEG2MPML,
+    MPEG2MPHL,
+    MPEG4HP41,
+    MPEG4HP41BD,
+    MPEG4HP422D,
+    MPEG4HP423D,
+    MPEG4HP42STEREO,
+    HEVCMP51,
+    HEVCM10P51,
 ]
+"""MPEG transfer syntaxes."""
+
+RLETransferSyntaxes = [RLELossless]
+"""RLE transfer syntaxes."""
 
-RLECompressedLosslessSyntaxes = [
-    RLELossless
+UncompressedTransferSyntaxes = [
+    ExplicitVRLittleEndian,
+    ImplicitVRLittleEndian,
+    DeflatedExplicitVRLittleEndian,
+    ExplicitVRBigEndian,
+]
+"""Uncompressed (native) transfer syntaxes."""
+
+# Deprecated
+JPEGBaseline = JPEGBaseline8Bit
```

Comment:
Should we also warn in `__getattr__` for all deprecated?

Reply:
Done